src/Services/DatabaseTools/ORMSqliteDatabaseTool.php

```diff
@@ -83,4 +90,38 @@ public function loadFixtures(array $classNames = [], bool $append = false): Abst
 
         return $executor;
     }
+
+    protected function disableForeignKeyChecksIfApplicable(): void
+    {
+        if (!$this->isSqlite()) {
```

Comment:
Do we need to check that we use a SQLite database? This class should be loaded only for SQLite.

Reply:
That's true, it's potentially unnecessary, I was following what the `ORMDatabaseTool` does. Happy to remove it though.